From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 28 Aug 2020 22:04:05 +0100
Subject: overlays: Fix error in README

spi0-cs has been renamed spi0-2cs, not spi0-1cs.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/README | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index a0adf476eafa..e2d90595c3c0 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -2430,7 +2430,7 @@ Params: cs0_pin                 GPIO pin for CS0 (default 8)
 
 
 Name:   spi0-cs
-Info:   This overlay has been renamed spi0-1cs, keeping spi0-cs as an
+Info:   This overlay has been renamed spi0-2cs, keeping spi0-cs as an
         alias for backwards compatibility.
 Load:   <Deprecated>
 
